src/WebJobs.Script/Utility.cs

```diff
@@ -996,6 +997,52 @@ public static FunctionAppContentEditingState GetFunctionAppContentEditingState(I
             }
         }
 
+        public static void GetScaleInstancesToProcess(
+            IEnvironment environment,
+            IFunctionsHostingConfiguration hostingConfiguration,
+            IEnumerable<IScaleMonitor> scaleMonitors,
+            IEnumerable<ITargetScaler> targetScalers,
+            out List<IScaleMonitor> scaleMonitrsToProcess,
+            out List<ITargetScaler> targetScalersToProcess)
+        {
+            scaleMonitrsToProcess = new List<IScaleMonitor>();
+            targetScalersToProcess = new List<ITargetScaler>();
+
+            string value = environment.GetEnvironmentVariableOrDefault(EnvironmentSettingNames.TargetBaseScalingEnabled, "0");
+
+            // Check if TBS enabled on stamp level
+            if (value == "1")
```

Comment:
`TARGET_BASED_SCALING_ENABLED` accepts "true" (case-insensitive), or "1" in scale controller -- please add this here!

Reply:
In [function](https://learn.microsoft.com/en-us/azure/azure-functions/functions-app-settings) for we use "1" as equivalent of "true".

for example:
https://learn.microsoft.com/en-us/azure/azure-functions/functions-app-settings#website_run_from_package

should we also use "true". TARGET_BASED_SCALING_ENABLED is not documented yet.

Reply:
Resolving this conversation via offline conversation with @pragnagopa. Ok to check if value == "1" only.